packages/status/src/class-status.php

```diff
@@ -113,6 +113,45 @@ public function is_single_user_site() {
 		return 1 === (int) $some_users;
 	}
 
+	/**
+	 * If the site is a local site.
+	 *
+	 * @since 8.8.0
+	 *
+	 * @return bool
+	 */
+	public function is_local_site() {
```

Comment:
Should we use this method up at L46? `$offline_mode = false === strpos( $site_url, '.' );`

Reply:
👍 Done in 6dcf0fa691aab9d1229b3fc089c3ce768d58cbe2